src/redux/combinedReducer.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import bookReducer from './books';
+
```

Comment:
**Optional Changes**
- [ ] Import `applyMiddleware` - to use some handy Redux middleware, like redux logger. 
This really helps allow during development to monitor the state of the application.